python/lesson_contributor_count.py

```diff
@@ -2,7 +2,7 @@
 import requests
 import json
 
-token = "GITHUB_TOKEN"
+token = GITHUB_TOKEN
```

Comment:
I _think_ you need:

```python
import os
token = os.environ["GITHUB_TOKEN"]
```

Reply:
Yup, I saw that after I tried pushing before.  Another build attempt is happening now. :-)

I thought that the existing `GITHUB_PAT` was invalid which is why I tried adding my own token.  If the existing `GITHUB_PAT` is valid, is it OK to keep just that one?  I believe it's tied to your account so I wanted to make sure first.

Reply:
Yes, you can use the existing GITHUB_PAT and delete the other one on Travis.